src/collectors/pe/decoder.cpp

```diff
@@ -310,19 +311,91 @@ void pe_decoder<_Bits>::get_imports(boost::filesystem::path const &module_path,
         //names are case unsensitive on windows!
         std::transform(dll_name.begin(), dll_name.end(), dll_name.begin(), ::tolower);
 
+		boost::filesystem::path full_path;
         // try to resolve full module path
         if (!find_module_path(module_path, dll_name)){
-            logging::log(logging::warning) << "couldn't find full path for module " << dll_name
+			logging::log(logging::warning) << "couldn't find full path for module " << dll_name
                 << " imported by module " << module_path << std::endl;
-        }
+			full_path = Env::root() / "./" / dll_name;
+			logging::log(logging::warning) << "dll name:" << full_path <<  std::endl;
```

Comment:
sotty about that, but looks like you're spacing is configured strangely. Mixing tab and spaces?

Reply:
"sed 's/ /\t/g' decoder.cpp > decoder.cpp" has done the job, but github will think that I have change the entire file